### ISSUE TITLE
Infrequent error "pop from empty list" when writing in Firefox

### ISSUE THREAD

**Issue by MarcoZehe** [nvaccess/nvda#11197]
### Steps to reproduce:

1. In Firefox, open Gmail and start composing a new message.
2. On purpose, add a few spelling mistakes so Firefox flags these. Note that you may have to have the spell checking dictionary installed for this if you don't already have it.

### Actual behavior:

Occasionally, NVDA plays the error sound and records this in the log:

```
ERROR - core.CorePump.run (08:07:05.178) - MainThread (19756):
errors in this core pump cycle
Traceback (most recent call last):
  File "core.pyc", line 517, in run
  File "braille.pyc", line 2216, in pumpAll
  File "braille.pyc", line 1952, in handlePendingCaretUpdate
  File "braille.pyc", line 1958, in _doCursorMove
  File "braille.pyc", line 1011, in update
  File "braille.pyc", line 849, in _addTextWithFields
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 342, in getTextWithFields
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 305, in _getText
IndexError: pop from empty list
```

### Expected behavior:

No such errors.

#### NVDA installed/portable/running from source:

Installed.

#### NVDA version:

version alpha-20220,1c4519a6

#### Windows version:

10.0 (19H2)

#### Name and version of other software in use when reproducing the issue:

Firefox, release or Nightly.

#### Other information about your system:

Freedom Scientific Focus 40 Blue 5th generation braille display connected.

#### Does the issue still occur after restarting your computer?

Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No, recently new bug in alphas.

#### If addons are disabled, is your problem still occuring?

Yes.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?

Yes.

CC @jcsteh

**Comment by Adriani90** [comment]
@MarcoZehe is this fixed now in NVDA's last alpha? Note there was PR #11318 which probably solved this now.

**Comment by jcsteh** [comment]
Closing as a duplicate of #10619, fixed by #11318. Marco, if you can still repro with latest alpha, please reopen.
